templates/cAdvisor/cadvisor.cfg

```diff
@@ -0,0 +1,1360 @@
+"/SENTRY/X/cadvisor/credentialsList" = { REPLACE = "" },
+"/SENTRY/X/cadvisor/databaseType" = { REPLACE = "" },
+"/SENTRY/X/cadvisor/description" = { REPLACE = "Docker Monitoring using cAdvisor================================This template provides monitoring of the docker containers and file systems using cAdvisor.* File System Statistics* Container statisticsThe template uses the cAdvisor REST API.* Add the Docker host under the hosts tab and define the system type as `Other`.* Specify the port on which the cAdvisor exposes metrics under the PORT macros." },
+"/SENTRY/X/cadvisor/disabled" = { REPLACE = "0" },
+"/SENTRY/X/cadvisor/embeddedFileList" = { REPLACE = "" },
+"/SENTRY/X/cadvisor/label" = { REPLACE = "CAdvisor" },
+"/SENTRY/X/cadvisor/macroList" = { REPLACE = "PORT;8080" },
+"/SENTRY/X/cadvisor/messageList" = { REPLACE = "" },
+"/SENTRY/X/cadvisor/parameterList" = { REPLACE = "CollectionErrorCount;0" },
+"/SENTRY/X/cadvisor/productVersion" = { REPLACE = "10300" },
+"/SENTRY/X/cadvisor/runAlertForEachCollectionError" = { REPLACE = "0" },
+"/SENTRY/X/cadvisor/shareCookies" = { REPLACE = "1" },
+"/SENTRY/X/cadvisor/systemType" = { REPLACE = "other" },
+"/SENTRY/X/cadvisor/thresholdList" = { REPLACE = "CollectionErrorCount;0 0 0 0 0 0;1 1 2147483646 0 0 2;0 0 0 0 0 0" },
+"/SENTRY/X/cadvisor/type" = { REPLACE = "Template" },
+"/SENTRY/X/cadvisor:containers/actionList" = { REPLACE = "" },
+"/SENTRY/X/cadvisor:containers/authenticationType" = { REPLACE = "" },
+"/SENTRY/X/cadvisor:containers/bodyType" = { REPLACE = "none" },
+"/SENTRY/X/cadvisor:containers/contentType" = { REPLACE = "application/x-www-form-urlencoded" },
+"/SENTRY/X/cadvisor:containers/headers" = { REPLACE = "" },
+"/SENTRY/X/cadvisor:containers/httpMethod" = { REPLACE = "GET" },
+"/SENTRY/X/cadvisor:containers/httpRequestContent" = { REPLACE = "textOnly" },
+"/SENTRY/X/cadvisor:containers/label" = { REPLACE = "Containers" },
+"/SENTRY/X/cadvisor:containers/messageList" = { REPLACE = "" },
+"/SENTRY/X/cadvisor:containers/parameterList" = { REPLACE = "ExecutionTime;0,HTTPStatusCode;0,Result;1,Status;1" },
+"/SENTRY/X/cadvisor:containers/parent" = { REPLACE = "cadvisor" },
+"/SENTRY/X/cadvisor:containers/pollingIntervalList" = { REPLACE = "httpRequestColl;continuous;120;" },
+"/SENTRY/X/cadvisor:containers/productVersion" = { REPLACE = "10300" },
+"/SENTRY/X/cadvisor:containers/reportAsCollectionError" = { REPLACE = "0" },
+"/SENTRY/X/cadvisor:containers/successfulHttpStatusCode" = { REPLACE = "200-208,226" },
+"/SENTRY/X/cadvisor:containers/thresholdList" = { REPLACE = "ExecutionTime;0 0 0 0 0 0;1 15 30 0 0 1;1 30 2147483646 0 0 2,Status;0 0 0 0 0 0;1 1 1 0 0 1;1 2 2 0 0 2" },
+"/SENTRY/X/cadvisor:containers/timeout" = { REPLACE = "30" },
+"/SENTRY/X/cadvisor:containers/type" = { REPLACE = "HTTPRequest" },
+"/SENTRY/X/cadvisor:containers/url" = { REPLACE = "http://%{HOSTNAME}:%{PORT}/api/v1.3/containers" },
```

Comment:
This template is used for monitoring docker container right?

We should use /api/v1.3/containers/docker if we want to compute docker container only:
_"Note that the root container (/) contains usage for the entire machine. All Docker containers are listed under /docker. Also, type=name is not required in the examples above as name is the default type."_ (https://github.com/google/cadvisor/blob/master/docs/api_v2.md)

Reply:
API v1.3 is better than v2 to get stats for each docker. I tried v2 and then went back to 1.3. Regarding Type, in a real user environment there may be different types. We cannot design the template suitable only to our env.